config/bidderinfo.go

```diff
@@ -246,6 +246,35 @@ func processBidderInfos(reader InfoReader, normalizeBidderName func(string) (ope
 			infos[string(normalizedBidderName)] = info
 		}
 	}
+	return processAliasBidderInfo(infos)
+}
+
+func processAliasBidderInfo(infos BidderInfos) (BidderInfos, error) {
+	for bidderName, bidderInfo := range infos {
+		if len(bidderInfo.AliasOf) > 0 {
+			if err := validateAliases(bidderInfo, infos, bidderName); err != nil {
+				return nil, err
+			}
+			parentBidderInfo := infos[bidderInfo.AliasOf]
```

Comment:
should we check if `parentBidderInfo` exist? considering scenario where alias adapter has mentioned incorrect parent adapter name

```
parentBidderInfo, ok := infos[bidderInfo.AliasOf]
if !ok {
   // return some error
}
```

Reply:
At this point of code, parentBidderInfo will exist since validateAliases function makes sure of it.